window.py

```diff
@@ -3,30 +3,25 @@
 #pyinstaller window.spec
 import PySimpleGUI as gui
 import logging
-from samase import Samase
+from samase import Samase, OpType
 import os
 
 LOGFILE = "sezam.log"
 
-def exists_localSamase(locals: dict) -> bool:
-    if "localSamase" not in locals:
-            print("Attempted to perform an operation without a directory selected!")
-            return False
-    else:
-        return True
+localSamase = None
+unpackSamase = None
 
-def exists_unpackSamase(locals: dict) -> bool:
-    if "unpackSamase" not in locals:
-            print("Attempted to perform an operation without a directory selected!")
-            return False
-    else:
-        return True
+def assertSet(value: any) -> bool:
```

Comment:
One function with meaningful param instead of two.

At first glance it seems that passing `locals()` as arg was redundant, since you can call `locals()` inside anyways. But maybe this is a quirk of python I am unaware of and locals gives vars from the current function scope and not the local file scope like I imagined, anyways super redundant if we can just check if the var is None.